europe-custom.yaml

```diff
@@ -242,7 +242,7 @@ tools:
     tool_panel_section_label: Extract Features
   - name: extract_genomic_dna
     owner: iuc
-    tool_panel_section_label: "Other Tools"
+    tool_panel_section_label: "Fetch Sequences / Alignments"
```

Comment:
With special characters it might be better to specify not the `tool_panel_section_label` but the `tool_panel_section_id`

Reply:
Reason is that Galaxy has some magic to convert those labels to IDs, but with special characters, you never know ...

Reply:
I see. With `tool_panel_section_id` as the yaml key, then?

Reply:
Yes: https://github.com/usegalaxy-eu/usegalaxy-eu-tools/blob/master/single-cell-ebi-gxa.yaml.lock#L12